crates/sui-swarm-config/src/network_config_builder.rs

```diff
@@ -50,6 +50,9 @@ pub struct ConfigBuilder<R = OsRng> {
     genesis_config: Option<GenesisConfig>,
     reference_gas_price: Option<u64>,
     additional_objects: Vec<Object>,
+    /// Building configs for deployment (local or in private testnet) rather than testing.
+    /// When this is true, we use more realistic parameters (especially for Narwhal).
```

Comment:
This comment seems to not match the name of the variable.

Reply:
Ok, I think I was confused by `local`, which I took for testing configuration. Not sure how other folks think, but maybe just `for_deploy` or something else without mentioning `local`.

Reply:
Changed it to the other way (for_testing), hope it's more clear now